fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/ambassador/simulation/ElectricVehicleUnit.java

```diff
@@ -63,6 +75,11 @@ protected boolean handleEventResource(Object resource, long eventType) {
             return true;
         }
 
+        if (resource instanceof ChargingStationDiscoveryResponse response) {
+            onCharginStationDiscoveryResponse((ChargingStationDiscoveryResponse) response);
```

Comment:
This cast is not necessary
```suggestion
            onCharginStationDiscoveryResponse(response);
```